qcore/srf.py

```diff
@@ -11,15 +11,19 @@
 
 from math import ceil, cos, floor, radians, sqrt, sin, degrees, atan
 from subprocess import Popen, PIPE
+import sys
 
 import numpy as np
 
 try:
     import alphashape
+    from alphashape import optimizealpha
```

Comment:
Why are we importing both the module and a specific function from the module? Should either be only the module or only the classes/functions we need

Reply:
Agreed